LibGAP.jl/src/conversion.jl

```diff
@@ -1,28 +1,33 @@
 ## Conversion to GAP
 
-to_gap(str :: String)         = MakeString(str)
-to_gap(v :: Any)              = v
-
-
-function to_gap( v :: Bool ) :: MPtr
-    if v
-        return True
-    else
-        return False
-    end
-end
+to_gap(v :: MPtr)       = v
+to_gap(v :: GapFFE)     = v
+to_gap(v :: String)     = MakeString(v)
+to_gap(v :: Bool)       = v ? True : False
 
 function to_gap(v :: Array{T, 1} where T) :: MPtr
     l = NewPlist(length(v))
     for i in 1:length(v)
         l[i] = v[i]
     end
+    # FIXME: size of l is never set, thus remains zero
```

Comment:
Actually, I just realized this complaint is wrong! Because you are using `l[i] = ...` defined below, which calls `ASS_LIST`, which *does* adjust the list size. So all should be fine. Of course best would be to verify this is accurate by adding a test which does something like `@test GAP.GAPFuncs.Length(to_gap([1,2,3])) == 3`.